packages/story-editor/src/app/media/local/useContextValueProvider.js

```diff
@@ -236,7 +237,8 @@ export default function useContextValueProvider(reducerState, reducerActions) {
         processVideoAudio(id, src);
       }
 
-      const imageSrc = type === 'image' ? src : poster;
+      const imageSrc =
+        type === 'image' ? getSmallestUrlForWidth(0, resource) : poster;
```

Comment:
Something for the future:

Do we still store `posterId` for resources?

If so, we could theoretically fetch `/wp-json/web-stories/v1/media/<posterId>` and get a smaller image size for poster images.